.env.sample

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=http://localhost:3000/api
```

Comment:
@codeshifu, great job in elegantly implementing this feature. However, a newline is needed here. Once again great job.